code/dqc/dfilechk.q

```diff
@@ -1,12 +1,14 @@
 \d .dqc
-dfilechk:{[tname;dirname]                                                                                       /- function to check .d file. Sample use: .dqe.dfilechk[`trade;getenv `KDBHDB]
-  system"l ",dirname;
+dfilechk:{[tname]                                                                                               /- function to check .d file. Sample use: .dqc.dfilechk[`trade]
+  .lg.o[`dfilechk;"Checking if two latest .d files match"];
   if[not `PV in key`.Q;
-    .lg.o[`dfilechk;"The directory is not partitioned"]; :0b];
+    .lg.o[`dfilechk;"The directory is not partitioned"];
+    :(0b;"The directory is not partitioned")];
   if[2>count .Q.PV;
-    .lg.o[`dfilechk;"There is only one partition, therefore there are no two .d files to compare"]; :1b];
+    .lg.o[`dfilechk;"There is only one partition"];
+    :(1b;"There is only one partition, therefore there are no two .d files to compare")];
   u:` sv'.Q.par'[`:.;-2#.Q.PV;tname],'`.d;
   $[0=sum {()~key x} each u;
```

Comment:
I think this condition is a little unclear, I think it might be more clear if you used the TorQ `.os.Fex` function to check for file existence:

```suggestion
  $[all .os.Fex each u;
```

(probably with a comment to clarify things for anyone unfamiliar with `.os.Fex` e.g. `/check all .d files exist`)